modules/fx/arrow-fx/src/test/kotlin/arrow/fx/IOTest.kt

```diff
@@ -187,56 +187,6 @@ class IOTest : UnitSpec() {
       }
     }
 
-    "should complete when running a pure value with runAsync" {
```

Comment:
These should be kept or updated to explain the behavior IMO

Reply:
👍 I'll revert these changes and we can keep them until we can fix them

Reply:
@pakoito I was just looking at these tests. None of these actually run, only 1 since only one of the `runAsync` tests calls `unsafeRunSync`.

I fixed and pushed them, but we should really get rid of these combinators. Super confusing and they don't do anything.